server/routes/admin.ts

```diff
@@ -36,5 +40,21 @@ export default function routes(controllers: Controllers, router: Router, service
     ],
   })
 
+  get(paths.admin.placementRequests.withdrawal.new.pattern, placementRequestWithdrawalsController.new(), {
+    auditEvent: 'ADMIN_NEW_PLACEMENT_REQUEST_WITHDRAWL',
+  })
+  post(paths.admin.placementRequests.withdrawal.create.pattern, placementRequestWithdrawalsController.create(), {
+    redirectAuditEventSpecs: [
```

Comment:
Worth an audit event for when the controller redirects to `paths.admin.placementRequests..show.pattern`?